src/bundle/Resources/views/themes/admin/content_type/list.html.twig

```diff
@@ -83,6 +84,11 @@
                 <td class="ez-table__cell">{{ content_type.identifier }}</td>
                 <td class="ez-table__cell">{{ content_type.id }}</td>
                 <td class="ez-table__cell">{{ content_type.modificationDate|ez_full_datetime }}</td>
+                <td class="ez-table__cell ez-table__cell--has-action-btns text-right">
```

Comment:
Wouldn't be possible to have the new button within the same column? That way it would look more consitent with other tables in the app